app/controllers/dashboard/current_tasks_controller.rb

```diff
@@ -95,19 +89,19 @@ def experiment_filter
     def prepare_due_date(task)
       if task.completed?
         return { state: '', text: I18n.t('dashboard.current_tasks.completed_on_html',
-                                         date: I18n.l(task.completed_on, format: :full_date)) }
+                                         date: I18n.l(task.completed_on, format: :full_date)).html_safe }
```

Comment:
Don't use html_safe

Reply:
Why not? Can't be injected anything from translations. We have some HTML marks in translations.

Reply:
Hm.. does not help with `_html` suffix. Probably because we are using translations inside of the helper method, not in erb view. Without html_safe `<b>` is rendered

Reply:
Move I18n render to views